tests/test_start_date.py

```diff
@@ -0,0 +1,149 @@
+import tap_tester.connections as connections
+import tap_tester.runner as runner
+from base import YotpoBaseTest
+from tap_tester.logger import LOGGER
+
+class YotpoStartDateTest(YotpoBaseTest):
+    """Instantiate start date according to the desired data set and run the test"""
+
+    start_date_1 = ""
+    start_date_2 = ""
+    streams = None
+
+    def name(self):
+        return "tap_tester_yotpo_start_date_test"
+
+    def test_run(self):
+        expected_streams = self.expected_streams() 
+        self.run_test("2021-09-09T00:00:00Z", "2022-06-20T00:00:00Z", expected_streams)
```

Comment:
If run_test is only used once then we can remove it and keep all the code in test_run() itself.